src/main/scala/path/Path.scala

```diff
@@ -16,6 +18,9 @@ import com.originate.scalypher.where.Where
 
 case class Path(start: Node, pieces: Seq[PathPiece] = Seq.empty) extends Referenceable {
 
+  def set(assignment: Assignment, rest: Assignment*): SetQuery =
```

Comment:
i think varargs is :+1: since you want that invariant. and not having that guarantee would make this way more complex, it seems